### PR TITLE
Docs 6681 feedback mutual tls not mentioned hn

### PR DESCRIPTION
**Pull request by hannanelson**
Need info on which endpoints use WebSockets.

### CODE REVIEW

**Review by bahui80 — APPROVED** [review]
LGTM

File: modules/ROOT/pages/rtm-agent-whitelists.adoc [hunk 1]
```diff
@@ -13,7 +13,23 @@ image:logo-server-active.png[link="/runtime-manager/deployment-strategies", titl
 
 In your network, you must whitelist the hostnames and ports of various parts of the Anypoint Platform to allow Runtime Manager agent in a customer-hosted Mule runtime engine to communicate with the online Anypoint Platform APIs and services managed by MuleSoft.
```
Comment (kbostrom) [inline modules/ROOT/pages/rtm-agent-whitelists.adoc]:
```suggestion
In your network, you must whitelist the hostnames and ports of various parts of Anypoint Platform to allow Runtime Manager agent in a customer-hosted Mule runtime engine to communicate with the online Anypoint Platform APIs and services managed by MuleSoft.
```

File: modules/ROOT/pages/rtm-agent-whitelists.adoc [hunk 2]
```diff
@@ -13,7 +13,23 @@ image:logo-server-active.png[link="/runtime-manager/deployment-strategies", titl
```
Comment (kbostrom) [inline modules/ROOT/pages/rtm-agent-whitelists.adoc:13]: INSTRUCTIONS FOR EDIT

a wee PR for your Monday enjoyment. Green only (you've seen this all before):

File: modules/ROOT/pages/rtm-agent-whitelists.adoc [hunk 3]
```diff
@@ -13,7 +13,23 @@ image:logo-server-active.png[link="/runtime-manager/deployment-strategies", titl
 
 In your network, you must whitelist the hostnames and ports of various parts of the Anypoint Platform to allow Runtime Manager agent in a customer-hosted Mule runtime engine to communicate with the online Anypoint Platform APIs and services managed by MuleSoft.
 
-These tables show you the ports or IP addresses/hostnames to add to your whitelists to allow communication between the agent and the Runtime Manager console:
+These tables show you the ports or IP addresses/hostnames to add to your whitelists to allow communication between the agent and the Runtime Manager console.
```
Comment (kbostrom) [inline modules/ROOT/pages/rtm-agent-whitelists.adoc]: Try to replace / with the most likely or precise of "and" or "or"
```suggestion
These tables show you the ports or IP addresses and hostnames to add to your whitelists to allow communication between the agent and the Runtime Manager console.
```

File: modules/ROOT/pages/rtm-agent-whitelists.adoc [hunk 4]
```diff
@@ -13,7 +13,23 @@ image:logo-server-active.png[link="/runtime-manager/deployment-strategies", titl
...
+
+Because the following endpoints use mutual TLS authentication, to establish the connection, you must configure SSL passthrough to allow the certificates:
```
Comment (kbostrom) [inline modules/ROOT/pages/rtm-agent-whitelists.adoc:18]: Is this okay: "to allow the certificates"?
authenticate?
enable?

**Review by kbostrom — APPROVED** [review]
Easy peasy!

**Pull request merged**